### ISSUE TITLE
Backup-BCContainerDatabases add possibility to set compression

### ISSUE THREAD

**Issue by stefanoPrimo** [microsoft/navcontainerhelper#1417]
PLEASE DO NOT INCLUDE ANY PASSWORDS OR TOKENS IN YOUR ISSUE!!!

**Describe the issue**
Usually if I backup a DB with SQL server the bak file is around 400 MB but If I export the DB via the command "Backup-BcContainerDatabases" the bak file is around 2 GB 

**Scripts used to create container and cause the issue**
```
New-BcContainer $containerName `
    -artifactUrl $artifactUrl `
	-alwaysPull `
    -accept_eula `
    -auth UserPassword `
	-Credential $credential `
    -licenseFile $LicenseFile `
    -updateHosts `
    -doNotExportObjectsToText `
    -shortcut None `
    -dns 8.8.8.8 `
    -doNotCheckHealth `
    -bakFile $BackFilePath
```

**Full output of scripts**
```
...
```

**Screenshots**

**Additional context**
- does it happen all the time?
- did it use to work?

**Comment by freddydk** [comment]
Backup with compression is not supported in SQL Express but I have added a compress parameter if you are using a non-express external SQL Server.

**Comment by freddydk** [comment]
Shipped in BcContainerHelper 1.0.13-preview256
